qemu/tests/cfg/multi_nics_hotplug.cfg

```diff
@@ -12,27 +12,32 @@
         - nic_8139:
             no ppc64 ppc64le
             pci_model = rtl8139
-            match_string = "8139"
             nics = ""
             extra_params += "-net none"
         - nic_virtio:
             pci_model = virtio-net-pci
-            match_string = "Virtio network device"
             Host_RHEL.m5:
                 pci_model = virtio
             nics = ""
             extra_params += "-net none"
         - nic_e1000:
             no ppc64 ppc64le
+            no RHEL.8
             pci_model = e1000
-            match_string = "Gigabit Ethernet Controller"
+            nics = ""
+            extra_params += "-net none"
+        - nic_e1000e:
```

Comment:
`e1000e` was introduced to qemu from `2.6.0`, so here should add param `required_qemu` to restrict the version to test, like: https://github.com/autotest/tp-qemu/blob/48ad741de8790be76fc162c92e3cd9bf0a525e8d/qemu/tests/cfg/qemu_img.cfg#L132

Reply:
@luckyh Updated it, thanks!